shared/kubernetes/mizuTapperSyncer.go

```diff
@@ -145,6 +148,10 @@ func (tapperSyncer *MizuTapperSyncer) watchTapperEvents() {
 					event.Reason,
 					event.Note))
 
+			if event.Reason == "Killing" {
```

Comment:
why should we have special treat for killing?
it will continue in the next if...
maybe  add additional info to debug log, like the event reason?

Reply:
The problem is that 'killing' reason causes the next if to get an error